package/MDAnalysis/core/topologyattrs.py

```diff
@@ -30,7 +30,6 @@
 from ..exceptions import NoDataError, SelectionError
 from .topologyobjects import TopologyGroup
 from . import selection
-from . import flags
```

Comment:
This was a double import. You can see the first import of flags if you look at the whole file.